test/passes/remove-unused-brs_enable-multivalue.txt

```diff
@@ -1669,6 +1669,109 @@
   )
   (unreachable)
  )
+ (func $br-to-table-initial-tee (param $a i32)
```

Comment:
How would you feel about porting this test to lit as a preliminary step?

Reply:
Hmm, I think the remaining unported tests had reasons they were not ported yet? Do you remember what reason this particular file had? I don't think I see anything obvious.

But if the test can be ported automatically then I'd rather do that later as I've already done the work to add a test in the current form.

Reply:
No, there are still plenty of tests to port automatically. I'll get that PR stream restarted. Porting the test before hand would make reviewing it marginally easier, but porting it later sounds fine to me.